shoes-core/lib/shoes/logger.rb

```diff
@@ -1,66 +1,47 @@
 # frozen_string_literal: true
+require 'logger'
+
 class Shoes
-  module Logger
-    class << self
-      def register(name, obj)
-        @loggers ||= {}
-        @loggers[name] = obj
-      end
+  class Logger
+    def initialize
+      @loggers = []
+    end
 
-      def unregister(name)
-        @loggers.delete(name)
-      end
+    def <<(logger)
+      @loggers << logger
+      self
+    end
```

Comment:
Decided that `LoggerCollection` was a little more straight-forward and plain, so ran with that. Thanks for the nudge on it @PragTob... happier with where it landed.